docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -36,7 +36,7 @@ You cannot update {dfanalytics-cap} configurations. Instead, delete the
 [[dfa-dataframe-size-limitations]]
 === {dataframe-cap} memory limitation
 
-{dfanalytics-cap} can analyze {dataframes} that fit into the memory limit 
```

Comment:
This limitation is a bit unclear.  Which setting does it pertain to (max_machine_memory_percent or max_model_memory_limit)?  What sort of error (if any) occurs if you try to use transforms that exceed this limit? Is it possible to determine in advance whether you're likely to hit this limitation?

Reply:
I'll address this in another PR.